spec/features/todo_list_crud_spec.rb

```diff
@@ -0,0 +1,80 @@
+require 'rails_helper'
+
+feature "Todo list", :type => :feature do
+
+  scenario "create to-do list" do
+    sign_in("tester@example.tld", "test-password")
+    click_link "Lists"
+
+    create_new_list("New test list")
+  end
+
+  scenario "edit to-do list" do
+    sign_in("tester@example.tld", "test-password")
+    click_link "Lists"
+
+    create_new_list("New test list")
+
+    expect(current_path).to eq(lists_path)
+    expect(page).to have_content("Edit")
+    click_link "Edit"
+
+    # 1. Get the last list item (naturally the list object created).
+    # 2. Pass in the list object from the above into the edit_list_path.
+    list = List.last                                 # [1]
+    expect(current_path).to eq(edit_list_path(list)) # [2]
+    expect(page).to have_content("Edit list")
+    fill_in "Title", with: "Updated list title"
+    click_button "Update list"
```

Comment:
Consider adding expectations that the resulting page does indeed have the updated list title.